tests/test_parser.py

```diff
@@ -0,0 +1,43 @@
+from home_assistant_bluetooth import BluetoothServiceInfo
+from sensor_state_data import DeviceClass, DeviceKey
+
+from ruuvitag_ble import RuuvitagBluetoothDeviceData
+
+OUTDOOR_SENSOR_DATA = (
+    b"\x05\x05\xa0`\xa0\xc8\x9a\xfd4\x02\x8c\xff\x00cvriv\xde\xad{?\xef\xaf"
+)
+INDOOR_SENSOR_DATA = (
+    b"\x05\x0e\xa4M~\xc8\x18\xfc\xbc\xfd\xf0\xff\xb4+\xf6\x00\x10<\xd97\x0f\xf7\xaa\x48"
+)
+
+KEY_TEMPERATURE = DeviceKey(key=DeviceClass.TEMPERATURE, device_id=None)
+KEY_HUMIDITY = DeviceKey(key=DeviceClass.HUMIDITY, device_id=None)
+KEY_PRESSURE = DeviceKey(key=DeviceClass.PRESSURE, device_id=None)
+KEY_VOLTAGE = DeviceKey(key=DeviceClass.VOLTAGE, device_id=None)
+KEY_MOVEMENT = DeviceKey(key="movement_counter", device_id=None)
+
+
+def bytes_to_service_info(payload: bytes) -> BluetoothServiceInfo:
+    return BluetoothServiceInfo(
+        name="Test",
+        address="00:00:00:00:00:00",
+        rssi=-60,
+        manufacturer_data={1177: payload},
+        service_data={},
+        service_uuids=[],
+        source="",
+    )
+
+
+def test_parsing_outdoor():
+    device = RuuvitagBluetoothDeviceData()
+    advertisement = bytes_to_service_info(OUTDOOR_SENSOR_DATA)
+    assert device.supported(advertisement)
+    up = device.update(advertisement)
+    expected_name = "RuuviTag DE:AD:7B:3F:EF:AF"
```

Comment:
Most parsers use a "short name" with the last 4 characters of the MAC. So "RuuviTag EFAF". 

You can get the last 4 characters with 

```python
from bluetooth_data_tools import short_address

identifier = short_address(service_info.address)
```

Reply:
I'll take a look at that (but we'll probably want to use the MAC address broadcast by the tag itself, especially since that always works, c.f. the UUID pseudo-addresses you get on macOS Core Bluetooth).

Thanks for the pointer!